io/eolearn/io/osm.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module containing tasks used for reading vector data from OpenStreetMap
+"""
+
+import overpass
+import geopandas
+from shapely.geometry import shape, box, mapping
+
+from eolearn.core import EOTask, FeatureType
+from sentinelhub.constants import CRS
+
+
+class OSMInput(EOTask):
+    """ Use OpenStreetMap (OSM) data from an Overpass API as input to a VECTOR_TIMELESS feature.
+    In case of timeouts or too many requests against the main Overpass endpoint, find additional
+    endpoints at see other options https://wiki.openstreetmap.org/wiki/Overpass_API#Public_Overpass_API_instances
+
+    :param feature_name: EOPatch feature into which data will be imported
+    :type feature_name: (FeatureType, str)
+    :param query: Overpass API Querystring: https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL
+    :type query: str
+    :param polygonize: Whether or not to treat ways as polygons, defaults to True
+    :type polygonize: bool
+    :param overpass_opts: Options to pass to the Overpass API constructor, see: https://github.com/mvexel/overpass-api-python-wrapper#api-constructor
```

Comment:
Code should pass pylint check, otherwise travis build will fail.
pylint: io/eolearn/io/osm.py:24:0: C0301: Line too long (149/120) (line-too-long)